src/components/Tombstone.vue

```diff
@@ -0,0 +1,171 @@
+<template>
+  <dl id="tombstone">
+    <!-- Registration Date -->
+    <template v-if="!!registrationDate">
+      <dt class="mr-2">Registration Date:</dt>
+      <dd id="registration-date">{{ registrationDate }}</dd>
+    </template>
+
+    <!-- Registration Number -->
+    <template v-if="!!registrationNumber">
+      <dt class="mr-2">Registration Number:</dt>
+      <dd id="registration-number">{{ registrationNumber }}</dd>
+    </template>
+
+    <!-- Business Number -->
+    <template v-if="!!businessNumber">
+      <dt class="mr-2">Business Number:</dt>
+      <dd id="business-number">{{ businessNumber }}</dd>
+    </template>
+
+    <!-- Incorporation Number -->
+    <template v-if="!!incorporationNumber">
+      <dt class="mr-2">Incorporation Number:</dt>
+      <dd id="incorporation-number">{{ incorporationNumber }}</dd>
+    </template>
+
+    <!-- NR Number -->
+    <template v-if="!!nameRequestNumber">
+      <dt class="mr-2">Name Request Number:</dt>
+      <dd id="name-request-number">{{ nameRequestNumber }}</dd>
+    </template>
+
+    <!-- Email -->
+    <template v-if="!!email">
+      <dt class="mr-2">Email:</dt>
+      <dd id="email" class="cursor-pointer" @click="editBusinessProfile()">
+        <span>{{ email }}</span>
+        <v-btn id="change-email-button" small text color="primary">
+          <v-icon small>mdi-pencil</v-icon>
+          <span>Change</span>
+        </v-btn>
+      </dd>
+    </template>
+
+    <!-- Phone -->
+    <template v-if="!!phone">
+      <dt class="mr-2">Phone:</dt>
+      <dd id="phone" class="cursor-pointer" @click="editBusinessProfile()">
+        <span>{{ phone }}</span>
+        <v-btn id="change-phone-button" small text color="primary">
+          <v-icon small>mdi-pencil</v-icon>
+          <span>Change</span>
+        </v-btn>
+      </dd>
+    </template>
+  </dl>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component } from 'vue-property-decorator'
+import { Getter } from 'vuex-class'
+import { navigate } from '@/utils'
+import { DateUtilities } from '@/services'
+
+@Component({})
+export default class Tombstone extends Vue {
+  @Getter getBusinessEmail!: string
+  @Getter getBusinessProfileUrl!: string
+  @Getter getBusinessNumber!: string
+  @Getter getFoundingDate!: Date
+  @Getter getFullPhoneNumber!: string
+  @Getter getIdentifier!: number
+  @Getter getNameRequest!: any
+  @Getter isFirm!: boolean
+
+  /** The Business ID string, if available. */
+  get businessId (): string {
+    return sessionStorage.getItem('BUSINESS_ID')
+  }
```

Comment:
What do you think about passing the `businessId` in as a prop?  If we eventually want to have the businessId passed to the view as route parameter, this makes this refactoring easier in the future.

Reply:
Good idea!

Reply:
Please see new commit and resolve this conversation if you are satisfied.